ldms/src/ldmsd/ldmsd_config.c

```diff
@@ -203,27 +203,28 @@ struct ldmsd_plugin_generic * load_plugin(const char *plugin_name,
 		}
 	}
 
-        plugin->name = strdup(plugin_name);
-        plugin->libpath = strdup(library_name);
+	plugin->name = strdup(plugin_name);
+	plugin->libpath = strdup(library_name);
 
 	return plugin;
 
 err_2:
-        dlclose(plugin->dl_handle);
+	dlclose(plugin->dl_handle);
 err_1:
-        free(plugin);
+	free(plugin);
 err_0:
 	return NULL;
 }
 
 static void unload_plugin(struct ldmsd_plugin_generic *plugin)
 {
-        assert(plugin != NULL);
-        free(plugin->name);
-        free(plugin->libpath);
-        plugin->api = NULL; /* about to be defunct when dlclose() is called */
-        dlclose(plugin->dl_handle);
-        free(plugin);
+	/* NB: do not call dlclose() as this will invalidate
```

Comment:
Please change note to make it clear this is temporary, and other bugs will be addressed to allow the dlclose() to be uncommented.